cv32e40p/tests/programs/corev-dv/corev_sanity_fp_zfinx_instr_test/corev-dv.yaml

```diff
@@ -14,6 +14,7 @@ plusargs: >
     +test_override_riscv_instr_stream=1
     +directed_instr_0=cv32e40p_float_zfinx_base_instr_stream,10
     +directed_instr_1=cv32e40p_fp_w_special_operands_instr_stream,5
+    +directed_instr_2=cv32e40p_fp_w_prev_rd_as_operand_instr_stream,5
```

Comment:
We need a separate test probably for this. Else too many things in 1 random test.

Reply:
This is your decision, but I _like_ tests with _lots_ of randomization.  If you think a separate test is necessary (for debugging, etc.), then perhaps create two tests: one with just `directed_instr_2` and another with all three.

Reply:
Thankyou for the advise. I will update accordingly in stage3 release